Ticket 104, Clarify the interpretation of scalar coordinate variables

painter1 opened the pull request:
This has all the changes of ticket 104, as described in comments 64 and 66.  Most of the changes had been in @reshel3 's branch #104 in her fork, but this branch has a couple very minor revisions, and comment 66 is newly implemented in this branch.

davidhassell commented:
The last sentence of the second paragraph of 5.7. Scalar Coordinate Variables: "Note however that use of this feature with a latitude, longitude, vertical, or time coordinate will inhibit COARDS conforming applications from recognizing them." seems to have gone missing ...

David

painter1 commented:
@davidhassell , Thank you for catching this!  I've fixed it (and converted a couple section names to actual cross-references) and will issue another pull request in a few minutes.
